README.rst

```diff
@@ -0,0 +1,22 @@
+################
+GRPC-OpenTracing
```

Comment:
Some of the python packages needed to be .rst to have the readme load on pypi... no reason that it's rst now, I just didn't change back (should I?).